src-tauri/src/internal_wallet.rs

```diff
@@ -187,6 +209,7 @@ impl InternalWallet {
             spend_public_key_hex: "".to_string(),
             config_path: Some(path.to_path_buf()),
             passphrase: None,
+            is_tari_address_generated: true,
```

Comment:
I thought we were going to change this to something easier to parse. Like `is_mining_address` ?